code/game/gamemodes/cult/ritual.dm

```diff
@@ -246,17 +226,25 @@ This file contains the arcane tome files as well as innate cultist emergency com
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
 		C.apply_damage(0.1, BRUTE, pick("l_arm", "r_arm"))
-		if("Call Forth The Geometer" == entered_rune_name)
+		if("Summon Nar-Sie" == entered_rune_name)
 			C.apply_damage(40, BRUTE, pick("l_arm", "r_arm"))
 			var/area/A = get_area(src)
 			var/locname = initial(A.name)
-			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual before it reaches a critical point.","Central Command Higher Dimensionsal Affairs")
-			if(!do_after(user, 500, target = get_turf(user)))
+			priority_announce("Figments from an eldritch god are being summoned by [user] into [locname] from an unknown dimension. Disrupt the ritual at all costs!","Central Command Higher Dimensionsal Affairs", 'sound/AI/spanomalies.ogg')
+			for(var/turf/B in orange (1, user))
+				var/obj/machinery/shield/N = new(B)
+				N.color = "red"
+				N.health = 60
+			if(!do_after(user, 400, target = get_turf(user)))
+				for(var/obj/machinery/shield/U in orange (1, user))
+					qdel(U)
 				return
 	if(!do_after(user, 50, target = get_turf(user)))
 		return
 	user.visible_message("<span class='warning'>[user] creates a strange circle in their own blood.</span>", \
 						 "<span class='cult'>You finish drawing the arcane markings of the Geometer.</span>")
+	for(var/obj/machinery/shield/S in orange (1, user))
+		qdel(S)
```

Comment:
This will delete any shields near you when you finish any rune, not just when you finish the nar-sie rune itself and not just the shields produced via the nar-sie rune.